src/styles/index.scss

```diff
@@ -1,3 +1,272 @@
+  gap: 100px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+@media (min-width: 1200px) {
+  .header {
+    max-width: 1200px;
+  }
+}
+
+.image {
+  display: block;
+}
+
+.logo__link {
+  width: 40px;
+  height: 40px;
+  align-items: center;
+}
+
+.nav {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+  gap: 20px;
+
+  &__link {
+    height: 60px;
+    font-weight: 500;
+    display: inline-flex;
+    align-items: center;
+    justify-content: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    color: #060b35;
+  }
+
+  &__link.is-active {
+    color: #00acdc;
+    position: relative;
+  }
+
+  &__list {
+    list-style: none;
+    color: #00acdc;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    gap: 20px;
+  }
+
+  &__item {
+    display: flex;
+  }
+  &__link:hover {
+    color: #00acdc;
+  }
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+
+  &::after {
+    content: '';
+    display: block;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    height: 4px;
+    border-radius: 4px;
+    background-color: #00acdc;
+  }
+}
+
+.stars {
+  display: flex;
+}
+
+.stars--hidden {
+  display: none;
+}
+
+.stars__star {
+  background-image: url(../images/star.svg);
+  background-position: center;
+  margin-right: 4px;
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0),
+.stars--1 .stars__star:nth-child(-n + 1),
+.stars--2 .stars__star:nth-child(-n + 2),
+.stars--3 .stars__star:nth-child(-n + 3),
+.stars--4 .stars__star:nth-child(-n + 4),
+.stars--5 .stars__star:nth-child(-n + 5) {
+  background-image: url(../images/star-active.svg);
+}
+
+p,
+h1 {
+  margin: 0;
+  padding: 0;
+}
+
+.card {
+  width: 200px;
+  height: 408px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  transition: all 0.3s ease;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 3px;
+
+    &:hover {
+      color: #34568b;
+      cursor: pointer;
+      transition: color 0.3s ease;
+    }
+  }
+
+  &__product {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    color: #616070;
+    margin-bottom: 18px;
+  }
+
+  &__price {
+    display: flex;
+    margin-bottom: 14px;
+    justify-content: space-between;
+
+    &-label {
+      font-size: 12px;
+      font-weight: 400;
+      height: 18px;
+      color: #616070;
+    }
+
+    &-value {
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+      color: #060b35;
+    }
+  }
+
+  &__button {
+    background-color: #00acdc;
```

Comment:
Use a created variable for repeated color everywhere 
<img width="320" alt="image" src="https://github.com/user-attachments/assets/31c5b93a-4004-4c32-8915-7668a652a6cc" />